ui/apps/platform/src/Containers/Policies/Table/PoliciesTablePage.tsx

```diff
@@ -10,10 +10,6 @@ import {
     AlertVariant,
     Divider,
     Button,
-    Flex,
-    Toolbar,
-    ToolbarContent,
-    ToolbarItem,
```

Comment:
not sure how these were passing through before 🤔

Reply:
Uh, oh. Maybe CI not running lint? There is also `console.log` in PolicyCategorySidePanel.tsx file

Reply:
Until we get to the bottom of it, run `yarn lint` in ui folder before you merge.